### PR TITLE
Changes win module background field to image node

### LINKED ISSUE #4804: Image node reference
in `dosomething_campaign` settings add a reference to a node id, this should follow the example of
the "Alternate header image nid" field

Then you can use the `dosomething_image` functions to pass the correct images to the theme

### PR DESCRIPTION
**Pull request by itsjoekent**
Fixes #4804 

**don't merge, need to test**

@angaither

### CODE REVIEW

**Comment by itsjoekent** [comment]
tested ready for review

File: lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc [hunk 1]
```diff
@@ -159,7 +159,8 @@ function dosomething_campaign_preprocess_win_module(&$vars, &$wrapper) {
   $rb_progress = dosomething_reportback_get_reportback_total_plus_override($node->nid);
   $copy_token_replaced = token_replace(variable_get('dosomething_campaign_win_copy'), array('node' => $node));
   $vars['win_copy'] = str_replace('[campaign_progress]', $rb_progress, $copy_token_replaced);
-  $background_fid = variable_get('dosomething_campaign_win_background_image');
+  $background_nid = (int) variable_get('dosomething_campaign_win_background_image');
+  $vars['background_landscape_url'] = dosomething_image_get_image_url($background_nid, 'landscape');
```
Comment (angaither) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc:163]: we probably don't want the landscape crop pulled in, @sbsmith86 do you know which crop you'll want?

Comment (sbsmith86) [inline lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc:163]: @angaither @deadlybutter we will probably need landscape and square for desktop vs mobile. Similar to how we handle the hero images on campaign pages.

**Comment by itsjoekent** [comment]
Added square crop, anything else needed? @sbsmith86

**Comment by sbsmith86** [comment]
@deadlybutter should be it! thanks!

**Comment by itsjoekent** [comment]
@angaither good to go then?

**Comment by angaither** [comment]
:+1:

**Pull request merged**